libraries/AP_Camera/AP_Camera_Servo.cpp

```diff
@@ -43,6 +49,28 @@ bool AP_Camera_Servo::trigger_pic()
     return true;
 }
 
+
+bool AP_Camera_Servo::set_zoom(ZoomType zoom_type, float zoom_value) {
```

Comment:
Hi there, thanks for this.  Small nitpick is in the cpp files we put the bracket on the next line